### DISCUSSION TITLE
Email display

### DISCUSSION THREAD

**Discussion by alexbrowng** [directus/directus#8538] [Feature Requests]
Create a display for fields containing an email that has an icon in the collection view that includes a "mailto" link.

The display options could be:

* Subject: Allows to indicate a default subject.
* Body: Allows to indicate the default body of the message.
* CC: Allows to indicate the recipient in copy

**Comment by rijkvanzanten** [comment]
I think this: https://github.com/directus/directus/discussions/8540#discussioncomment-1425411 applies here as well

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
